oeps/processes/oep-0055-project-maintainers.rst

```diff
@@ -0,0 +1,309 @@
+the platform.
+
+In order to ensure that the project is sustainable for the long term,
+maintenance must be shared by the entire community.  The project must
+be robust enough that a it can tolerate firms and contributors leaving
+the ecosystem.
+
+As a project we must acknowledge that maintenance is hard work, but
+work that must be done for the platform to be fit for use.  The best
+way of ensuring this is to have a named person or persons who are
+responsible for getting the work done.
+
+Specification
+=============
+
+Definitions
+-----------
+
+The term **repository** will mean a repository on the GitHub social
+coding platform.
+
+The term **component** will typically mean the unit of software housed
+in a single GitHub repository.  Exceptions will be made in the case of
+edx-platform where there may be multiple distinguishable components in
+that single repository.  Components will typically, but not always, be
+hosted at the Open edX GitHub organization.  Components will be "part
+of" or "critical to" the open-source platform.
+
+The term **maintainer** will mean a named person responsible for the
+health of component and the component's community.  The remainder of
+this specification will be focused on the details of the role.
+
+The term **core contributor** will mean a named person participating
+in the `Core Contributor Program`_ and meeting the requirements of
+that program.
+
+The term **maintainer scrum-of-scrums** will mean group of all
+maintainers who are responsible for flagging issues to project
+leadership and proposing process improvements for component
+maintenance.
+
+The term **Technical Oversight Committee** (TOC) will mean the group
+responsible for the overall oversight and stewardship of the Open edX
+project. The TOC itself was established and is governed by a formal
+charter.  You can read the full document `here
+<https://openedx.org/wp-content/uploads/2021/12/TCRIL-Technical-Oversight-Committee-Charter-rev.11-16.21.pdf>`_.
+
+Context
+-------
+Historically, the Open edX project has not had a well-defined role
+of maintainer.  At times, community members stepped into this role
+informally.  The role was also part of a broader "owner" role at edX.
+Owners were also responsible for the health of the component they
+owned both in the repository and in production; owners answered the
+pager when something went wrong.
+
+The role of the repository or project maintainer is both a community
+focused and a technical role[#]_.  Maintainers will be the public face
+of each component and should be welcoming and available.
+
+Non-technical responsibilities of maintainers include:
+
+* Effective communication with community members, other maintainers,
+  and project leadership
+* Disciplined management to ensure that maintenance work is done
+* Enforcement of the project's standards
+* The ability to solve or escalate issues facing users of the
+  component
```

Comment:
I could potentially see an explicit requirement of kindness/friendliness/openness. And collaboration, and (maybe) welcoming new CCs to their repo.

Perhaps - but I'm not sure how I feel about this - some sort of explicit commitment along the lines of the Declaration of Commitment to the CC program

Reply:
I like this idea.  I think that calling out kindness/friendliness/openness as valued behaviors makes sense.  We would review the Code of Conduct and ensure we feel that comes through strongly enough there. 

I also like the idea the the maintainer is expected to by kind, but also firm.  Kindness isn't not enforcing the standards, or not rejecting a troubled PR because you don't want to hurt feelings.

The Declaration of Commitment feels a little different though I could see having one for maintainers.  

@hurtstotouchfire I wanted to make sure I grok what part of Sarina's comment you are supporting.

Reply:
> I also like the idea the the maintainer is expected to by kind, but also firm. Kindness isn't not enforcing the standards, or not rejecting a troubled PR because you don't want to hurt feelings.

+1 yes, my thoughts exactly